nata/styles.black.css

```diff
@@ -0,0 +1,42 @@
+body {
```

Comment:
The idea of having another CSS is to use custom properties and override them here. We don't need any CSS rules here.